pkg/usagestats/reporter.go

```diff
@@ -33,13 +32,31 @@ const (
 	defaultStatsServerURL      = "https://stats.grafana.org/mimir-usage-report"
 )
 
+const (
+	CustomInstallationMode  = "custom"
+	HelmInstallationMode    = "helm"
+	JsonnetInstallationMode = "jsonnet"
+)
+
 type Config struct {
-	Enabled bool `yaml:"enabled" category:"experimental"`
+	Enabled          bool   `yaml:"enabled" category:"experimental"`
+	InstallationMode string `yaml:"installation_mode" category:"experimental"`
 }
 
 // RegisterFlags adds the flags required to config this to the given FlagSet
 func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.BoolVar(&cfg.Enabled, "usage-stats.enabled", true, "Enable anonymous usage reporting.")
+	f.StringVar(&cfg.InstallationMode, "usage-stats.installation-mode", CustomInstallationMode, "Installation mode (one of: custom, helm, jsonnet).")
```

Comment:
It's probably better to use the corresponding constants, to avoid drift:
```suggestion
	f.StringVar(&cfg.InstallationMode, "usage-stats.installation-mode", CustomInstallationMode, fmt.Sprintf("Installation mode (one of: %s, %s, %s).", CustomInstallationMode, HelmInstallationMode, JsonnetInstallationMode))
```